include/deal.II/numerics/vector_tools_boundary.h

```diff
@@ -96,7 +96,15 @@ namespace VectorTools
    * Thus, the elements in the component mask corresponding to the components
    * of these non-primitive shape functions must be @p false.
    *
-   * See the general documentation of this namespace for more information.
+   * @note This function expects the same ComponentMask for all boundary
+   * indicators passed to this function. In case of multiple boundary
+   * indicators with different ComponentMask, this function needs to be called
+   * multiple times. For performance reasons, it might be reasonable to use
```

Comment:
Let's write this as follows:
```suggestion
   * @note This function applies the same ComponentMask to all parts of
   * the boundary indicated by keys of the `function_map` argument.
   * If you want to apply different component masks to parts of the boundary
   * represented by different boundary indicators, this function needs to be called
   * multiple times. For performance reasons, it might be reasonable to use
```